app/models/article.rb

```diff
@@ -1,30 +1,49 @@
 class Article < ApplicationRecord
+
   include PgSearch
   multisearchable :against => [:title, :summary, :content]
+
   extend FriendlyId
   friendly_id :title, use: :slugged
+
+  # Sets how many articles are shown per page; this is aprt of the kaminari
+  # gem's pagination config.
+  paginates_per 10
+
   belongs_to :section, optional: true
 
   has_many :authorships
   has_many :contributors,
            through: :authorships,
            dependent: :destroy,
            source: :user
-
   has_many :media, dependent: :destroy
   has_many :comments, dependent: :destroy
   has_many :outquotes, dependent: :destroy
 
   def init
     self.update(is_published: false)
+
+    if self.summary.nil? or self.summary.summary.empty?
```

Comment:
This is fine here, but be very careful about `or` vs `||` in Ruby

Reply:
https://stackoverflow.com/questions/2083112/difference-between-or-and-in-ruby

Reply:
Thanks for the reference.